examples/evm/seth.py

```diff
@@ -190,6 +190,9 @@ def create_contract(self, owner, balance=0, init=None, address=None):
         with self.locked_context('seth') as context:
             assert context['_pending_transaction'] is None
         assert init is not None
+        # bytecode is only valid in 32-byte segments
```

Comment:
Do you have any reference for this? I'm failing to find that.

Reply:
> Definition: len(a) is the number of bytes in a binary string a. The type of len(a) is assumed to be uint256.
from the [ABI](https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI) specification.

I took this to mean that properly encoded instructions would each take at least one uint256 in memory (32-bytes). After re-reading, I'm sure I misread. All the ABI encoded stuff I have seen has instructions padded out to 32 bytes, though.

After thinking about it, this still doesn't solve the underlying problem, since encoding invalid data may be a feature for Manticore (thinking fuzzing like @ggrieco-tob is doing). Thoughts?